docs/whatsnew/v0-5-0.rst

```diff
@@ -16,8 +16,8 @@ API changes
 New features
 ############
 
-* Add `inactivity_costs` as an option for `Flow`s. Inactivity costs is a cost for times where a Flow is not operated.
-* It is now possible to optimise the size of `Flow`s that have a status variable. Internally, this is done by the `InvestNonConvexFlowBlock` which provides the possibility to perform capacity optimization of assets considering their min/max loads, as well as their operation status.
+* Add `inactivity_costs` as an option for `Flow`\s. Inactivity costs is a cost for times where a Flow is not operated.
+* It is now possible to optimise the size of `Flow`\s that have a status variable. Internally, this is done by the `InvestNonConvexFlowBlock` which provides the possibility to perform capacity optimization of assets considering their min/max loads, as well as their operation status.
```

Comment:
I experienced a warning when I tried to render the docs: "WARNING: Inline interpreted text or phrase reference start-string without end-string.". The backslash was a solution I found online: https://stackoverflow.com/questions/58633196/restructuredtext-inline-code-followed-by-s-results-in-inline-interpreted-text, but reading through it again, it seems to be discouraged.

If I remember correctly, even the respective test failed (I have no working tox setup currently, but tried using the CI).

Long story short: It was rather a quick fix. I would be fine to remove it again in case the tests do allow to do so.

Reply:
As "\s" is sometimes read as space, I really want to avoid those. I'd prefer a workaround like in:
> `Flow` objects

Reply:
Yes, thank you. I agree with that and introduced the suggested workaround.